docs/sketch-build-process.md

```diff
@@ -157,8 +157,8 @@ its cloud-based nature may make the locations of libraries less obvious.
 
 ## Compilation
 
-Sketches are compiled by avr-gcc and avr-g++ according to the variables in the boards.txt file of the selected board's
-[platform](platform-specification.md).
+Sketches are compiled by platform-specific versions of `gcc` and `g++` according to the variables in the boards.txt file
+of the selected board's [platform](platform-specification.md).
```

Comment:
```suggestion
Sketches are compiled by architecture-specific versions of `gcc` and `g++` according to the variables in the boards.txt
file of the selected board's [platform](platform-specification.md).
```
I feel that the use of the term "platform" with two separate meanings within the same sentence is potentially confusing.

Reply:
Good catch. Thanks!